src/azure-cli/setup.py

```diff
@@ -139,7 +139,8 @@
     'jsmin~=2.2.2',
     'jsondiff==1.2.0',
     'packaging~=20.9',
-    'PyGithub==1.38',
+    'PyGithub~=1.38',
+    'PyNaCl~=1.4.0',
```

Comment:
you also need to add dependencies in requirementxxx.txt

Reply:
These already existed in requirementxxx.txt